selfdrive/ui/.gitignore

```diff
@@ -12,3 +12,4 @@ qt/setup/setup
 qt/setup/reset
 qt/setup/wifi
 qt/setup/updater
+translations/generated.h
```

Comment:
can we give it a more specific file name or will it grow to include more than just offroad alerts?

Reply:
y, can you suggest a name?

Reply:
We will also add alert events to this file later